docs/windows.md

```diff
@@ -0,0 +1,65 @@
+# DC/OS on windows
+
+DC/OS on Windows is a **work in progress**. As such there is **no guarantee it works**.  It is not currently complete. This document covers where the project is and what is planned for the near future.
+
+## The high-level goal of DC/OS on Windows
+
+Allow a full DC/OS installation as it is on Linux today, but with the addition of public and private Windows agents.
+Once installed, jobs can be pushed to either Linux or Windows agent, using necessary constraints as generally things that work on one of these platforms is unlikely to work on the other. Constraints today include things like the node being public or private, but with windows additions that will need to be a part of the constraints also.
+There are currently no plans to make the masters run on Windows.
+
+## How to install
+
+As of the writing of this document, the only way of installing DC/OS is using [acs-engine](https://github.com/Azure/acs-engine/blob/master/docs/dcos.md). This document only talks about Linux, although [this acs-engine .json configuration file](https://github.com/Azure/acs-engine/blob/master/examples/windows/dcos-winagent.json) does show an example of configuration for 1 master and 2 windows agents. Once installed, specifying a service constraint of `os=windows` would allow a service to be sent to a windows agent.
+
+This installation approach allows you to deploy a specific version of DC/OS in an Azure environment with a set of masters, and a set of public and private Linux/Windows agents. The Windows agents are installed from a *last known good* version of the windows binaries.
+
+The goal is to extend the advanced DC/OS install methods of deployment, such that there would be a Windows and Linux bootstrap node which allows Linux masters to be deployed, along with both Linux and Windows agents.
+
+An extended goal is to extend our current acs-engine model to allow the deployment of these bootstrap nodes and to deploy the masters and agents automatically from these bootstrap nodes.
+
+## Building DC/OS packages for Windows
```

Comment:
Does this section still apply? It sounds like it's describing the work done in this PR as future work.

Reply:
kind of, although I deleted part of it. It says how things are build for deployment. This differs from Linux where pkgpanda builds things as part of deployment